src/start_server.ts

```diff
@@ -100,11 +100,17 @@ export function getApp(options: ServerOptions) {
 
   app.use('/components/', polyserve);
 
+  const isFileRequest = (filePath: String) => {
+    return ['html', 'js', 'json', 'css', 'png', 'jpg', 'jpeg', 'gif'].filter((ext) => {
```

Comment:
Can we make this a bit more flexible? I don't like the idea of only supporting and maintaining a subset of all possible file types.

If we want to return a 404 for all paths that include a file extension, can you look into a solution using either regex matching or something with `lastIndexOf(".")`?